index.js

```diff
@@ -112,13 +141,33 @@ console.log(array1.fill(6));
     - Besides the methods on Person.prototype, babies have the ability to `.play()`:
         + Should return a string "Playing with x", x being the favorite toy.
 */
-function Baby() {
+const Baby=function (name,age,favoriteToy) {
   this.name=name;
   this.age=age;
```

Comment:
The way we subclass another object that you have created is by using the call method, in this case we would call:
```js
function Baby(name, age, favoriteToy) {
  Person.call(this, name, age);
  this.favoriteToy = favoriteToy;
}